features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx

```diff
@@ -124,39 +124,52 @@ function TryOutController(props) {
         let keys;
         let selectedKeyTypes = 'PRODUCTION';
         let accessToken;
-        const promiseSubscriptions = restApi.getSubscriptions(apiID);
-        promiseSubscriptions.then((subscriptionsResponse) => {
-            if (subscriptionsResponse !== null) {
-                subscriptionsList = subscriptionsResponse.obj.list.filter((item) => item.status === 'UNBLOCKED'
-                || item.status === 'PROD_ONLY_BLOCKED');
+        if (!api.lifeCycleStatus && api.lifeCycleStatus.toLowerCase() === 'prototyped') {
```

Comment:
shouldn't the subscription call happen if api.lifeCycleStatus.toLowerCase() !== 'prototyped') ?

Reply:
fixed issues